### DISCUSSION TITLE
blog/1

### DISCUSSION THREAD

**Discussion by giscus** [AnuragGowda/nextjs-blog#1] [Announcements]
# blog/1

An introductory exploratory paper of the basics Neural Networks. Uses mathematics of forward and backwards propagation to build a Neural Network to recognize digits from scratch.

http://localhost:3000/blog/1

**Comment by SquareEggplant** [comment]
![me rn](https://media1.tenor.com/m/71DeLT3bO0AAAAAd/dead-chat-dead-chat-skeleton.gif)
